server/index.js

```diff
@@ -1,16 +1,29 @@
 import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+
 import users from './routes/users';
 import boards from './routes/boards';
 
-import { PORT } from './configs/config';
+import { PORT, MONGO_CONNECTION_STRING } from './configs/config';
 
 const app = express();
 
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
 app.use('/users/', users);
 app.use('/boards/', boards);
 
 app.get('/', (req, res) => {
   res.send('Hey!');
 });
 
+mongoose.connect(MONGO_CONNECTION_STRING, { useNewUrlParser: true }, (err) => {
+  if (err) {
```

Comment:
please throw the error here, instead of console.logging, because probably you dont want to start the server when the mongo connection is not successful